src/frontend/src/pages/CreateWorkPackagePage/CreateWorkPackageFormView.tsx

```diff
@@ -221,12 +222,12 @@ const CreateWorkPackageFormView: React.FC<CreateWorkPackageFormViewProps> = ({ a
           </Grid>
         </Grid>
         <Box display="flex" gap={2} sx={{ mt: 2 }}>
-          <SubmitButton variant="contained" color="primary" type="submit" disabled={!allowSubmit}>
+          <NERSuccessButton variant="contained" color="primary" type="submit" disabled={!allowSubmit}>
             Create
-          </SubmitButton>
-          <Button variant="outlined" color="secondary" onClick={onCancel}>
+          </NERSuccessButton>
+          <NERFailButton variant="outlined" color="secondary" onClick={onCancel}>
             Cancel
-          </Button>
+          </NERFailButton>
```

Comment:
cancel should be on the left, and create should be on the right. Also they should both be in the bottom right of the page to be consistent with the rest of these types of buttons